src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
@pathikrit, this repo is intended to be a rewrite of Scala collections.
Collections that will be included in this rewrite can't depend on `scala.collection.mutable`.
If you rely on this functionality, you'll need to add it to the new collection infrastructure present in this repo.